bench/src/bulk.rs

```diff
@@ -51,27 +62,72 @@ async fn server(mut incoming: quinn::Incoming) -> Result<()> {
     let quinn::NewConnection {
         mut uni_streams, ..
     } = handshake.await.context("handshake failed")?;
-    let mut stream = uni_streams
-        .next()
-        .await
-        .ok_or_else(|| anyhow!("accepting stream failed"))??;
-    trace!("stream established");
+
+    loop {
+        let mut stream = match uni_streams.next().await {
+            None => return Ok(()),
+            Some(Err(quinn::ConnectionError::ApplicationClosed(_))) => return Ok(()),
+            Some(Err(e)) => return Err(e).context("accepting stream failed"),
+            Some(Ok(stream)) => stream,
+        };
+        trace!("stream established");
+
+        let _: tokio::task::JoinHandle<Result<()>> = tokio::spawn(async move {
+            let mut n = 0u64;
+            while let Some((data, offset)) = stream.read_unordered().await? {
+                n = n.max(offset + data.len() as u64);
+            }
```

Comment:
What are we keeping track of `n` for?

Reply:
leftover from the old version. Will remove